events/elections/2020/README.md

```diff
@@ -161,6 +161,8 @@ the [Steering Committee Election Charter]
 |                    Name                    | Organization/Company |                        GitHub                        |
 |:------------------------------------------:|:--------------------:|:----------------------------------------------------:|
 | [Jane Containerface](./biotemplate.md)     |      ExampleCo       | [@github](https://github.com)                        |
```

Comment:
Should remove this line, but that will fall out as the set of these are merged/rebased.

/lgtm